emcc.py

```diff
@@ -1801,7 +1803,8 @@ def check_human_readable_list(items):
 
       # Bitcode args generation code
       def get_clang_command(input_files):
-        args = [clang_compiler] + newargs + input_files
+        cflags = shared.COMPILER_OPTS + shared.emsdk_cflags(newargs)
```

Comment:
for anything to work before, these would have had to be added somewhere, and then we can remove that place in this PR. I see that for `COMPILER_OPTS` but not for the other flags, so I'm missing something?

Reply:
Yes, in this PR COMPILER_OPTS is not longer used internally.  Its still there just because it can be set in the `.emscripten` file.. but really I think we should rid of it completely in a followup.

In any case COMPILER_OPTS used have all the sdk flags added to it in tools/shared.py.   That is no longer the case.